src/peft/tuners/reft/config.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2023-present the HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass, field
+from typing import List, Optional, Union
+
+from peft.tuners.lycoris_utils import LycorisConfig
+from peft.utils import PeftType
+
+
+@dataclass
+class LoReftConfig(LycorisConfig):
+    """
+    This is the configuration class to store the configuration of a [`LoReftModel`].
+
+    Args:
+        r (`int`): LoReFT rank.
+        alpha (`int`): LoReFT alpha.
+        loc (`Optional[str]`):
+            Token locations are applied LoReFT.
```

Comment:
This needs much better explanation, ideally with a few examples. Also, it should be mentioned that this only really makes sense for language models, right?